airbyte-cdk/python/airbyte_cdk/models/airbyte_protocol.py

```diff
@@ -97,6 +98,14 @@ class Config:
     failure_type: Optional[FailureType] = Field(None, description="The type of error")
 
 
+class AirbyteConfigMessage(BaseModel):
```

Comment:
Should we include `Connector` in the name to make it clear that this is a connector configuration message and not some other type of configuration that could be sent?  I suppose that since this is in the protocol and that is to allow communication between connectors and the platform, you could infer that it is for a connector.  I don't feel strongly one way or another -- I just want to make sure that we don't end up having to rename this at some point.

Reply:
I'm fine with that! I'll update this PR and the docs shortly.

Reply:
Updated via f5af5a2d15e2c02769d95d3a2e43110a1bf8daa8